kafka-native-test-container/src/test/java/com/ozangunalp/kafka/test/container/KafkaNativeContainerIT.java

```diff
@@ -76,12 +75,14 @@ void checkProduceConsume(KafkaNativeContainer container, Map<String, Object> con
     void verifyClusterMembers(KafkaNativeContainer container, Map<String, Object> configs, int expected) {
         try (KafkaCompanion companion = new KafkaCompanion(container.getBootstrapServers())) {
             companion.setCommonClientConfig(configs);
-
-            DescribeClusterResult describeClusterResult = companion.getOrCreateAdminClient().describeCluster();
-            await().atMost(30, TimeUnit.SECONDS).untilAsserted(() -> assertThat(describeClusterResult.nodes().get().size()).isEqualTo(expected));
+            await().atMost(30, TimeUnit.SECONDS).untilAsserted(() -> assertThat(getClusterSize(companion)).isEqualTo(expected));
```

Comment:
defect was here, I was querying a value object, rather than redescribing the cluster.